wagtailmodeladmin/helpers.py

```diff
@@ -189,6 +189,7 @@ def __init__(self, model, permission_helper, user,
         self.permission_helper = permission_helper
         self.inspect_view_enabled = inspect_view_enabled
         self.model_name = force_text(self.opts.verbose_name).lower()
+        self.model_name_plural = force_text(self.opts.verbose_name_plural).lower() if self.opts.verbose_name_plural else self.model_name
```

Comment:
@Luperi As far as I'm aware, `verbose_name_plural` is always populated in a model's `_meta', whether defined in the model's `Meta` class or not. So, this should be enough:

`self.verbose_name_plural = force_text(self.opts.verbose_name_plural).lower()`